ydb/core/kqp/executer_actor/kqp_literal_executer.cpp

```diff
@@ -173,6 +173,7 @@ class TKqpLiteralExecuter {
         NDqProto::TDqTask protoTask;
         protoTask.SetId(task.Id);
         protoTask.SetStageId(task.StageId.StageId);
+        protoTask.SetEnableSpilling(false); // TODO: enable spilling
```

Comment:
Will this flag enable spilling in channels? Or is this change only for comp nodes?

Reply:
No, it's only for comp nodes. For channels we have a flag in TChannel protobuf.